documentation/content/buckets.html

```diff
@@ -5,31 +5,9 @@
 
 <p>
 The content layer splits the document space into chunks called <em>buckets</em>.
-This document is about how documents map to buckets
-and how buckets are distributed and managed.
-Learn more about the <a href="idealstate.html">distribution algorithm</a>.
 </p><p>
-The various operations done through and within the content layer need some
-unit of data to operate on. This could be documents directly, but then the
-number of units existing can not be controlled by the content layer. Mapping
-documents to buckets, the number of units existing can be controlled.
-</p><p>
-Advantages of many units:
-<ul>
-  <li>Easy to create an even distribution</li>
-  <li>Issues detected are more specific,
-    relating to a smaller part of the document corpus</li>
-  <li>Operations on small units finish quickly</li>
-</ul>
-Advantages of few units:
-<ul>
-  <li>Per unit metadata takes up less space,
-    simplifying caching and metadata transfers.</li>
-  <li>Iterating metadata for all units is faster.</li>
-</ul>
-To strike a balance, documents are mapped to a set of buckets,
-where the number of buckets can be configured.
-As clusters grow or shrink, or document sizes in the cluster change,
+Documents are mapped to buckets.
+As clusters grow or shrink, or document counts change,
```

Comment:
Could consider emphasising that all this neat stuff happens without any required intervention. E.g.:
> Documents are mapped to buckets. The cluster automatically splits and joins buckets to maintain an even distribution across all nodes and to keep bucket sizes within configurable limits.

Reply:
agreed!